### ISSUE TITLE
Archive an unread article

### ISSUE THREAD

**Issue by erixtekila** [wallabag/wallabag#1126]
In order to archive an article, I've had to first mark it as read.
Maybe it'd be good to alert the user of the new status of the article too ?!

**Comment by tcitworld** [comment]
I don't really understand your first sentence, but the second one is good.

**Comment by erixtekila** [comment]
LOL @tcitworld Je peux le refaire en français si tu préfères ;)
Afin d'archiver un article, il m'a semblé qu'il fallait d'abord le marqué comme "lu".
Est-ce normal ? En tous cas, ce n'est pas explicite… d'où ma seconde phrase, qui a été comprise.

**Comment by tcitworld** [comment]
En fait archiver un article = le marquer comme lu.

**Comment by erixtekila** [comment]
Not in my experience. I had first to mark it as read, then archive it.
But, yes, I agree totally if archiving means having read it before.

**Comment by tcitworld** [comment]
I don't understand what you mean by archive an entry. 
When you mark an article as read, it is then found into the Archive category.

**Comment by erixtekila** [comment]
Ah sorry my bad ! I meant favorite, not archived…
Let me rephrase : in order to star an article, you have to first mark it as read. 
Sorry.

**Comment by tcitworld** [comment]
Strange... You should be able to star an article without marking it as read. Will look.

**Comment by j0k3r** [comment]
@erixtekila I've tested on the 1.9.0 and didn't got any problem to star an article from the unread list.
When you rollover an article on the unread page, did you see the little star or not ?

![image](https://cloud.githubusercontent.com/assets/62333/6882976/ad63a8c8-d599-11e4-9995-f414cdb90b31.png)

**Comment by erixtekila** [comment]
@j0k3r Yep. thanks.

**Comment by j0k3r** [comment]
Well, I've no idea what we did, but if you are now able to star an article from the unread page, it's great :)

**Comment by erixtekila** [comment]
ghost help ! ;)
